.github/workflows/dependabotifications.yml

```diff
@@ -26,4 +26,14 @@ jobs:
         run: gh pr merge --auto --merge "$PR_URL"
         env:
           PR_URL: ${{github.event.pull_request.html_url}}
-          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
\ No newline at end of file
+          GH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+    auto-approve:
```

Comment:
lets add a condition: so if ${{secrets.SSE_BOT_PAT}} is empty/not configured that this job is not executed